rules-reviewed/quarkus/springboot/springboot-web-to-quarkus.windup.groovy

```diff
@@ -0,0 +1,57 @@
+package quarkus.narayana
+
+import org.jboss.windup.config.GraphRewrite
+import org.jboss.windup.config.metadata.TechnologyReference
+import org.jboss.windup.config.operation.iteration.AbstractIterationOperation
+import org.jboss.windup.config.query.Query
+import org.jboss.windup.config.query.QueryPropertyComparisonType
+import org.jboss.windup.graph.model.FileLocationModel
+import org.jboss.windup.graph.model.WindupConfigurationModel
+import org.jboss.windup.graph.model.resource.FileModel
+import org.jboss.windup.graph.service.GraphService
+import org.jboss.windup.graph.service.WindupConfigurationService
+import org.jboss.windup.reporting.category.IssueCategory
+import org.jboss.windup.reporting.category.IssueCategoryRegistry
+import org.jboss.windup.reporting.config.Hint
+import org.jboss.windup.reporting.config.Link
+import org.jboss.windup.rules.apps.java.condition.SourceMode
+import org.ocpsoft.rewrite.context.EvaluationContext
+
+final IssueCategory mandatoryIssueCategory = new IssueCategoryRegistry().getByID(IssueCategoryRegistry.MANDATORY)
+
+ruleSet("springboot-web-to-quarkus-groovy")
```

Comment:
@m-brophy maybe some spacing between code blocks would make the rule easier to read